models/intel/text-image-super-resolution-0001/description/text-image-super-resolution-0001.md

```diff
@@ -0,0 +1,50 @@
+# text-image-super-resolution-0001
```

Comment:
Please also update `license.txt` to include this model.

Reply:
Done